lib/csv_importable/type_parser/zip_type_parser.rb

```diff
@@ -0,0 +1,17 @@
+module CSVImportable
+  class TypeParser::ZipTypeParser < TypeParser
+    def parse_val
+      not_digits unless value.count('0-9') == value.length
+      value = '0' + value until value.length >= 5
+      value
```

Comment:
For now, this is specific to the US. This is also assuming that if someone enters a zip + 4 (that has the dash), it appears the CSV does not drop the leading zeros, so we're only worried about zip values that are fewer than 5 characters.